### DISCUSSION TITLE
Concurrency-limited workflow runs should not notify when currently pending jobs are cancelled

### DISCUSSION THREAD

**Discussion by myitcv** [community/community#50725] [Actions]
### Select Topic Area

Bug

### Body

With reference to [the concurrency key docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency), specifically the second sentence in bold:

> When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be pending. **Any previously pending job or workflow in the concurrency group will be canceled**. To also cancel any currently running job or workflow in the same concurrency group, specify `cancel-in-progress: true`.*

We are seeing pending jobs in the concurrency group being cancelled. Per the docs, this is exactly as expected. 

However, we are getting notifications every time such a job is cancelled. 

This is creating an incredible amount of noise for what is documented and hence expected behaviour.

The notifications seem to suggest that something bad has happened, when in fact it is expected behaviour. Such is the volume of notifications we are receiving, genuine problems that we need to investigate are getting lost.

Please can you turn off these notifications?

_* I should note that I don't agree with this behaviour as a default. That point is well covered in https://github.com/orgs/community/discussions/41518, https://github.com/actions/runner/issues/1523, https://github.com/actions/runner/issues/2227, https://github.com/orgs/community/discussions/5435, and other discussions/issues etc._

**Comment by TomOConnor95** [comment]
Yep the email alerts for concurrency cancelled runs are pretty annoying!

**Comment by codeflorist** [comment]
i agree.

i have `Only notify for failed workflows` enabled in my profile and still get notified in these cases.

this is not a `failed workflow`, but an expected behaviour when using concurrency.

**Comment by snprajwal** [comment]
Is there an update on this? Maybe a workaround to silence it? I second the fact that this is SUPER annoying and adds a lot of noise

**Comment by SunJiFengPlus** [comment]
This bug remains unresolved to this day, i want notify when failed, not when cancelled.
